### DISCUSSION TITLE
What are all the cli options?

### DISCUSSION THREAD

**Discussion by frank8922** [gitpod-io/openvscode-server#211] [Q&A] [answered]
The main readme says to check to the docs branch for documentation however all I see is guides for running this on <insert cloud provider>. I was meerly trying to run this locally and was struggling to find the cli options for running the standalone server, the only ones I managed to find was if I was running it in a docker container. Where would I find all of the cli options for natively running it? They don't seem to be listed anywhere.

**Comment by akosyakov** [comment, marked as answer]
You can find the definition here: https://github.com/gitpod-io/openvscode-server/blob/4575b989a97668904eb76fcc8a25665c5958eecf/src/vs/server/serverEnvironmentService.ts#L12-L116
